Program/internal_exam2.md

```diff
@@ -0,0 +1,112 @@
+### PROGRAM NO:-8 {Create table cricketer(cid, cname, match, run) and insert 8 players records. Print player records with average.  and 
+### write this data into player.csv file. Do all this task from python}.
+
+
+```python
+import csv
+```
+
+
+```python
+import sqlite3 as sq
```

Comment:
why you import sqlite3 as " sq " ??

Reply:
Sqlite3 is bit lengthy to write again and again in program  therefore I used sq which is short to write and also to understand.
Instead of sq any word can be used, it can be of single character or any lengthy word

Reply:
I used " sq " as an alias in my code 
any valid name can be used as alias .